assets/src/hooks/useDetour.ts

```diff
@@ -79,9 +79,9 @@ export const useDetour = ({ routePatternId, shape }: OriginalRoute) => {
     detourShape.result && isOk(detourShape.result)
       ? detourShape.result.ok.directions
       : undefined
-
-  // Only append direction "Regular Route" after detour is finished
-  if (!detourShape.isLoading && directions && finishedDetour) {
+  // // Only append direction "Regular Route" after detour is finished
+  // if (!detourShape.isLoading && directions && finishedDetour) {
+  if (directions && finishedDetour) {
```

Comment:
This is the sort of thing that's tricky enough to test that I could be persuaded to add the `!detourShape.isLoading` condition back in untested at least for now, but I want to think a bit harder about what kind of test or tests might lead to that condition mattering.

Reply:
Going to add this back in and follow up with improving the coverage [in a later ticket](https://app.asana.com/0/1152340551558956/1207337422166099/f)

Reply:
[Feedback: Restore check for !detourShape.isLoading ](https://github.com/mbta/skate/pull/2580/commits/e851bd6b972ee3688aacca0efd8fac3ea752cba2)